src/test/ui/lint/lint-min-warn-cannot-allow2.rs

```diff
@@ -0,0 +1,11 @@
+// run-pass
+
+// FIXME(centril): enforce the warnings and such somehow.
```

Comment:
what does the fixme here mean? Are you saying you want the diagnostic to point to the `#![warn(warnings)]` in addition to talking about the minimum level of the lint?

Reply:
If you look at the `.stderr` file you'll see warnings being emitted; however, because it is a `// run-pass` test I have no way of enforcing those in the `.rs` file. Basically this needs changes to compiletest so that it can be enforced, see https://github.com/rust-lang/rust/issues/59659.